### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by namanrai**
# Frontend Fixes

<ol>
    <li>Multiple Fixes for the frontend</li>
    <li>In the Frontend Directory</li>
        <ul>
            <li>\merchStore\frontend\src\components</li>
        </ul>

Fixes:
    <ul>
           <li>Fixed the Navbar overflow</li>
           <li>Fixed the cross in the Navbar</li>
           <li>Fixed margin and overflow in the Footer</li>
           <li>Fixed the overflow in the signup popup</li>
           <li>Fixed the text overflow in the Admin Page</li>
           <li>Removed Address component from the Admin Page</li>
           <li>Converted Footer.css to MUI components</li>
    </ul>
    
</ol>

### CODE REVIEW

File: frontend/src/components/Footer.js [hunk 1]
```diff
@@ -29,37 +32,146 @@ export default function Footer(){
...
-        <div className='content'>
-            <div className='socials'>
-                <div className='footer-head'>
+    <Box sx={{aspectRatio: "100 / 12.5",
+              backgroundColor: '#1f1e3a',
+              [theme.breakpoints.down('tablet')]: {
+                aspectRatio: '100/25'
+            }}}>
+
+        <Box sx={{  paddingTop: '30px',
```
Comment (DROFEL) [inline frontend/src/components/Footer.js:41]: Ideally you would want to put styles in separate .js file in styles folder and then import that object but its also fine

**Review by DROFEL — APPROVED** [review]
LGTM!

**Review by ShafinMahmudCS — CHANGES_REQUESTED** [review]
Please resolve conflicts

**Comment by namanrai** [comment]
I have fixed the merge conflicts but the development branch has errors right now. Do you want me to merge right now or when that's fixed?

**Review by ShafinMahmudCS — APPROVED** [review]

**Pull request merged**